pdf/config.json5

```diff
@@ -378,6 +410,11 @@
                 },
               },
             },
+            stderr: {
+              '^.*$': {
+                nameGroup: 'ERROR'
```

Comment:
I'm more fine with this one not having special annotation because it's all caps.